experiments/calibration/run_calibration.jl

```diff
@@ -0,0 +1,124 @@
+using Distributed
+import ClimaCalibrate as CAL
+using ClimaCalibrate
+using ClimaAnalysis
+import ClimaAnalysis: SimDir, get, slice, average_xy
+import ClimaComms
+import EnsembleKalmanProcesses: I, ParameterDistributions.constrained_gaussian
+
+# Ensure ClimaComms doesn't use MPI
+ENV["CLIMACOMMS_CONTEXT"] = "SINGLETON"
+ClimaComms.@import_required_backends
+
+single_member_dims = (1,)
+function CAL.observation_map(iteration)
+    G_ensemble = Array{Float64}(undef, single_member_dims..., ensemble_size)
+
+    for m in 1:ensemble_size
+        member_path = CAL.path_to_ensemble_member(output_dir, iteration, m)
+        simdir_path = joinpath(member_path, "model_config/output_active/clima_atmos")
```

Comment:
I don't think you need to go into the `model_config/` directory here, but I'm not sure what `member_path` returns so it may be correct
```suggestion
        simdir_path = joinpath(member_path, "output_active/clima_atmos")
```

Reply:
I think the `model_config` comes from the name of the config file (`model_config.yml`). I could change this with the `job_id`, it would probably be good to keep a consistent job_id for calibration runs.

Reply:
If `job_id = model_config` this path will be right, but I'm not sure `job_id` is being set (the default is nothing)

Reply:
The job_id gets set to the name of the YAML file, I don't think it is ever `nothing` after the `get_coupler_config_dict` func:
`    job_id = isnothing(job_id) ? string(split(split(config_file, '/')[end], '.')[1]) : job_id`

https://github.com/CliMA/ClimaCoupler.jl/blob/main/experiments/ClimaEarth/user_io/arg_parsing.jl#L26 

Since this is only manually tested I would like to run the calibration once a week, should I make a new pipeline or do you think we could put this in the longruns?

Reply:
Ahh yes you're right, I missed that line.

Let's add it to the longruns since we already have a lot of coupler pipelines